publication-commons/src/main/java/no/unit/nva/publication/permissions/file/ClaimedChannelFileDenyStrategy.java

```diff
@@ -0,0 +1,41 @@
+package no.unit.nva.publication.permissions.file;
+
+import static no.unit.nva.publication.model.business.publicationchannel.ChannelType.PUBLISHER;
+import java.util.Optional;
+import no.unit.nva.model.FileOperation;
+import no.unit.nva.publication.model.business.FileEntry;
+import no.unit.nva.publication.model.business.Resource;
+import no.unit.nva.publication.model.business.UserInstance;
+import no.unit.nva.publication.model.business.publicationchannel.ClaimedPublicationChannel;
+
+public class ClaimedChannelFileDenyStrategy extends FileStrategyBase implements FileDenyStrategy {
+
+    public ClaimedChannelFileDenyStrategy(FileEntry file, UserInstance userInstance, Resource resource) {
+        super(file, userInstance, resource);
+    }
+
+    @Override
+    public boolean deniesAction(FileOperation permission) {
+        return isWriteOrDelete(permission) && hasClaimedPublisher() && isDeniedUser();
```

Comment:
Could this make trouble for external clients?

Reply:
Godt poeng, tillater ekstern client i første iterasjon.